start-tests.sh

```diff
@@ -355,10 +355,23 @@ if [[ "$SKIP_UI_TEST" == false ]]; then
   fi
 
   log_color "cyan" "Running console cypress tests."
-  # NOTE: This reduces intermittent failures due to the console not being ready.
-  #       In the future we should replace the sleep with a check for the console being ready.
-  echo "Waiting 60 seconds for console to be ready..."
-  sleep 60; 
+
+  echo "Waiting up to 10 minutes for console pods to reach Running status"
+  RUNNING="false"
+  ATTEMPTS=0
+  MAX_ATTEMPTS=20
+  INTERVAL=30
```

Comment:
You could reduce the interval to 5-10 seconds. I don't think it will create any performance issues and it could help the test to start faster.